pkg/chain/local/tbtc.go

```diff
@@ -627,6 +626,16 @@ func (tlc *TBTCLocalChain) Logger() *localChainLogger {
 	return tlc.logger
 }
 
+func (tlc *TBTCLocalChain) RandomSigningGroup(size int) []common.Address {
+	signers := make([]common.Address, size)
+
+	for i := range signers {
+		signers[i] = generateAddress()
+	}
+
+	return signers
+}
+
```

Comment:
Thoughts on keeping it in `local.go` next to  `generateAddress` and not as a function of `TBTCLocalChain`?